.github/workflows/_rust.yml

```diff
@@ -19,23 +20,29 @@ jobs:
         # TODO: https://github.com/rust-lang/cargo/issues/5220
         include:
           - runs-on: ubuntu-22.04
-            packages: # Intentionally blank as a package catch-all linter
+            packages: linux-rust-packages.sh
           - runs-on: macos-14
-            packages: -p connlib-client-apple -p snownet
+            packages: macos-rust-packages.sh
           - runs-on: windows-2022
-            packages: -p connlib-client-shared -p firezone-gui-client -p snownet
+            packages: windows-rust-packages.sh
```

Comment:
I refactored these because they were written in 3 places and I think already drifting out of sync

Reply:
Thanks for doing that!

What do you think of modifying the `.github/actions/setup-rust` action to just export the packages as a variable depending on runner.os instead?

That would keep CI configuration out of `scripts/`

Reply:
https://docs.github.com/en/actions/creating-actions/creating-a-composite-action#creating-an-action-metadata-file

```
outputs:
  packages:
    description: Relevant packages for the current OS
    value: ${{
      (runner.os == 'Windows' && '...') ||
      (runner.os == 'Linux' && '...') ||
      (runner.os == 'macOS' && '...') }}
```

Reply:
Yeah that might make it easier, and it would still show up in `git grep`. I'll try it.

Reply:
You might need single-quoted strings... I think GitHub might be quote-sensitive in some cases